scripts/review/diff.ts

```diff
@@ -0,0 +1,118 @@
+import util from 'node:util';
+import fs from 'node:fs/promises';
+import parseDiff from 'parse-diff';
+import assert from 'node:assert';
+
+type Options = {
+  diffPatchFileName: string;
+  verbose: boolean;
+};
+
+type Deployments = { [chainId: number]: string | string[] };
+
+function parseOptions(): Options {
+  const options = {
+    diffPatchFileName: { type: 'string' },
+    verbose: { type: 'boolean' },
+  } as const;
+  const { values } = util.parseArgs({ options });
+
+  const option = 'diffPatchFileName';
+  if (values[option] === undefined) {
+    throw new Error(`missing --${option} flag`);
+  }
+
+  return {
+    ...(values as Options),
+    verbose: values.verbose === true,
+  };
+}
+
+function hasChangeTypes(changes: parseDiff.Chunk['changes'], types: string[]): boolean {
+  return changes.length === types.length && changes.every(({ type }, i) => type === types[i]);
+}
+
+async function main() {
+  const options = parseOptions();
+  const debug = (...msg: unknown[]) => {
+    if (options.verbose) {
+      console.debug(...msg);
+    }
+  };
+
+  // Read the content of the files
+  const diffPatchContent = await fs.readFile(options.diffPatchFileName, 'utf-8');
+  const diffPatch = parseDiff(diffPatchContent);
+
+  let highestChainID = false;
```

Comment:
I hadn't noticed this before - Personally, I prefer camelCase with lower-case for acronyms (so `Id`, `Http`).

This is about my suggestion :point_up:, I think consistency matters more than what we chose - so feel free to go back to `chainID` or change `ID` -> `Id` in other places as well.

Reply:
Done 👍🏾